Packs/FeedGreyNoiseIndicator/Integrations/GreyNoiseIndicator/GreyNoiseIndicator.yml

```diff
@@ -0,0 +1,116 @@
+---
+category: Data Enrichment & Threat Intelligence
+commonfields:
+  id: GreyNoise Indicator Feed
+  version: -1
+configuration:
+- defaultvalue: 'true'
+  display: Fetch indicators
+  name: feed
+  type: 8
+  required: false
+- display: Username
+  name: credentials
+  type: 9
+  required: false
+- additionalinfo: Indicators from this integration instance will be marked with this reputation
+  defaultvalue: Suspicious
+  display: Indicator Reputation
+  name: feedReputation
+  options:
+  - None
+  - Good
+  - Suspicious
+  - Bad
+  type: 18
+  required: false
+- additionalinfo: Reliability of the source providing the intelligence data
+  defaultvalue: B - Usually reliable
+  display: Source Reliability
+  name: feedReliability
+  options:
+  - A - Completely reliable
+  - B - Usually reliable
+  - C - Fairly reliable
+  - D - Not usually reliable
+  - E - Unreliable
+  - F - Reliability cannot be judged
+  required: true
+  type: 15
+- additionalinfo: Select which GreyNoise Feed to ingest
+  defaultvalue: Malicious
+  display: GreyNoise Feed Type
+  name: greynoiseFeedType
+  options:
+  - Benign
+  - Malicious
+  - Benign + Malicious
+  - All
+  required: true
+  type: 15
+- additionalinfo: Supports CSV values.
+  display: Tags
+  name: feedTags
+  type: 0
+  required: false
+- additionalinfo: The Traffic Light Protocol (TLP) designation to apply to indicators fetched from the feed
+  display: Traffic Light Protocol Color
+  name: tlp_color
+  defaultvalue: GREEN
+  options:
+  - RED
+  - AMBER
+  - GREEN
+  - WHITE
+  type: 15
+  required: false
+- defaultvalue: indicatorType
+  display: ''
+  name: feedExpirationPolicy
+  options:
+  - never
+  - interval
+  - indicatorType
+  - suddenDeath
+  type: 17
+  required: false
+- defaultvalue: '10080'
+  display: ''
+  name: feedExpirationInterval
+  type: 1
+  required: false
+- defaultvalue: '1440'
+  display: Feed Fetch Interval
+  name: feedFetchInterval
+  type: 19
+  required: false
+- additionalinfo: When selected, the exclusion list is ignored for indicators from this feed. This means that if an indicator from this feed is on the exclusion list, the indicator might still be added to the system.
+  display: Bypass exclusion list
+  name: feedBypassExclusionList
+  type: 8
+  required: false
+- display: Use system proxy settings
+  name: proxy
+  type: 8
+  required: false
+- display: Trust any certificate (not secure)
+  name: insecure
+  type: 8
+  required: false
+description: "GreyNoise is a cybersecurity platform that collects and analyzes Internet-wide scan and attack traffic. With this integration, users can contextualize existing alerts, filter false-positives, identify compromised devices, and track emerging threats. This Integration provides a feed of IPv4 Internet Scanners from GreyNoise."
+display: GreyNoise Indicator Feed
+name: GreyNoise Indicator Feed
+script:
+  commands:
+  - arguments: []
+    description: Gets the feed indicators.
+    name: greynoise-get-indicators
```

Comment:
Do you know what the outputs of the command are?
For more details on outputs, please refer to https://xsoar.pan.dev/docs/integrations/context-and-outputs.

Reply:
So, this command generates what I think is the standard table for an indicator feed in the war room, and those indicators get added to the TIM.  I'm not sure what outputs would be captured here.  I could use some guidance on this one.